infra/machines/vcsa/create-esxi-vm.sh

```diff
@@ -17,19 +17,22 @@
 
 set -o pipefail
 
-echo "Checking govc version...$(govc version -require 0.7.1 2>&1)"
-
 usage() {
-    echo "Usage: $0 [-d DISK_GB] [-m MEM_GB] [-i ESX_ISO] [-s] ESX_URL VM_NAME" 1>&2
-    exit 1
+    cat <<'EOF'
+Usage: $0 [-d DISK_GB] [-m MEM_GB] [-i ESX_ISO] [-s] ESX_URL VM_NAME
+
+GOVC_* environment variables also apply, see https://github.com/vmware/govmomi/tree/master/govc#usage
+If GOVC_USERNAME is set, it is used to create an account on the ESX vm.  Default is to use the existing root account.
+If GOVC_PASSWORD is set, the account password will be set to this value.  Default is to use the given ESX_URL password.
+EOF
 }
 
 disk=48
 mem=16
 iso=VMware-VMvisor-6.0.0-3634798.x86_64.iso
-vib=http://download3.vmware.com/software/vmw-tools/esxui/esxui-signed-3843236.vib
+vib=http://download3.vmware.com/software/vmw-tools/esxui/esxui-signed-3976049.vib
 
-while getopts c:d:i:m:s flag
+while getopts c:d:hi:m:s flag
```

Comment:
The colon is only for flags that take an argument, '-h' and '-s' do not take an argument.